### PR TITLE
Add running instructions

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+It defines what test configuration to run; empty value runs all configurations.
+
+## Starting environment with Docker Compose
+
+```sh
+make env-up DB=mongodb
+```
+
+That command will start MongoDB in Docker container.
+Please note that running `make dance DB=ferretdb` after that would run tests against that MongoDB, but results would be compared against results expected for FerretDB.
```
Comment (seeforschauer) [inline CONTRIBUTING.md:27]: how to run correctly after running `make env-up DB=mongodb`? 
`make dance DB=ferretdb`, not `make dance DB=ferretdb TEST=ferret`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
+Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience, but not required.
+In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host as it does not involve Docker images building or PostgreSQL restarts.
+
+## Running tests
+
+```sh
+make dance DB=ferretdb TEST=ferret
```
Comment (seeforschauer) [inline CONTRIBUTING.md:10]: what parameters `DB` and `TEST` mean? 

here if assume `DB` - to be compared to and `TEST` - what is being tested, it is confusing that they both have `ferret` in word. I thought, DB=mongo and TEST=ferret.

File: CONTRIBUTING.md (hunk 1 above)
Comment (AlekSi) [reply CONTRIBUTING.md:27]: Not sure I understand the question

File: CONTRIBUTING.md (hunk 2 above)
Comment (AlekSi) [reply CONTRIBUTING.md:10]: That's explained below

File: CONTRIBUTING.md (hunk 1 above)
Comment (seeforschauer) [reply CONTRIBUTING.md:27]: > [](https://github.com/seeforschauer)Please note that running `make dance DB=ferretdb` after that would run tests against that MongoDB, but results would be compared against results expected for FerretDB.

Can't catch what it means..

File: README.md [hunk 3]
```diff
@@ -3,5 +3,12 @@
 [![Dance](https://github.com/FerretDB/dance/actions/workflows/dance.yml/badge.svg)](https://github.com/FerretDB/dance/actions/workflows/dance.yml)
 
 FerretDB integration testing tool named after [Ferret war dance](https://en.wikipedia.org/wiki/Weasel_war_dance).
+It runs integration tests of various software that uses MongoDB (such as [MongoDB Go driver](https://github.com/mongodb/mongo-go-driver), [Mongoose ODM](https://mongoosejs.com), etc) against both MongoDB and FerretDB, and compares results with ones expected by tests configurations.
+It is expected that most or all tests pass when run against MongoDB, so we mark a few or none tests as expected failures or skips in configuration.
+More tests fail (and are marked as expected failures in tests configuration) when run against FerretDB, but their number goes down over time.
 
-Work-in-progress.
+Dance also includes one additional set of integration tests (called `ferret`) that is written by FerretDB developers.
```
Comment (AlekSi) [inline README.md:10]: I can see how a test set name `ferret` can be confused with a database name / set's configuration `ferretdb`, but I did not like any alternative names I could come up with: `our`, `own`, `self`, etc. Any ideas?

(For example, here is MongoDB Go driver's configuration for FerretDB: https://github.com/FerretDB/dance/blob/d72807d789121b160ddcda91b75d1254b345af1c/tests/mongo-go-driver.yml#L15)

File: CONTRIBUTING.md (hunk 1 above)
Comment (AlekSi) [reply CONTRIBUTING.md:27]: Added more info to README below

File: README.md (hunk 3 above)
Comment (seeforschauer) [reply README.md:10]: origin?

File: CONTRIBUTING.md (hunk 1 above)
Comment (seeforschauer) [reply CONTRIBUTING.md:27]: now it's clear, thank you

File: README.md (hunk 3 above)
Comment (AlekSi) [reply README.md:10]: homebrew?

Comment (seeforschauer) [reply README.md:10]: homebrew +1

**Review by seeforschauer — APPROVED** [review]

**Pull request merged**